### PR TITLE
[FAB-35] Manage classroom

### PR DESCRIPTION
**Pull request by daystram**

### CODE REVIEW

**Review by nicolauscg — COMMENTED** [review]
looked at changes except Hub and classroom related containers, looks good

File: fableous-fe/src/containers/ClassroomListPage.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+  const handleCreate = () => {
+    setCreating(true);
+  };
+
+  const handleCancel = () => {
+    setCreating(false);
+  };
+
+  useEffect(() => {
```
Comment (nicolauscg) [inline fableous-fe/src/containers/ClassroomListPage.tsx:61]: whats the purpose of this useEffect?

Comment (daystram) [reply fableous-fe/src/containers/ClassroomListPage.tsx:61]: enforces classroom list to be refetched upon component reload

Comment (nicolauscg) [reply fableous-fe/src/containers/ClassroomListPage.tsx:61]: so that when going back to list page after editing classroom name, classroom list is refeched

**Review by nicolauscg — APPROVED** [review]
all changes reviewed, looks good!

**Pull request merged**